### DISCUSSION TITLE
Icon request: fa-milk-and-tomato (grocery)

### DISCUSSION THREAD

**Discussion by GaboDot** [FortAwesome/Font-Awesome#21217] [Icon Requests]
Hi.

I was looking for an icon that represents the products catalog, actually something like grocery icon, I got 2 pics of my ideas but it would be awesome if they were icons, this will be used on mobile menu of my site, also could be used to represent the "shop page" or "products page" on selling sites.

Here's my site: https://www.mandadoapp.com
I attach 2 pics.

Thank you!

![13_grocery_products-256](https://cloud.githubusercontent.com/assets/8822582/24261022/3a2d266e-0fbb-11e7-8b78-20b72a9b9620.png)
![food-vegeterian-food-icon](https://cloud.githubusercontent.com/assets/8822582/24261023/3a2e25a0-0fbb-11e7-8e66-50d29d96561c.png)

**Comment by aethior** [comment]
Related? ~~Duplicate?~~ #9213

**Comment by sensibleworld** [comment]
I think of this request as a couple of grocery items, whereas the other would have items in a grocery bag.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
